packages/forms-web-app/__tests__/unit/routes/before-you-start/enforcement-notice.test.js

```diff
@@ -0,0 +1,34 @@
+const { get, post } = require('../router-mock');
+const enforcementNoticeController = require('../../../../src/controllers/before-you-start/enforcement-notice');
+const fetchExistingAppealMiddleware = require('../../../../src/middleware/fetch-existing-appeal');
+const { validationErrorHandler } = require('../../../../src/validators/validation-error-handler');
+const {
+  rules: enforcementNoticeValidationRules,
+} = require('../../../../src/validators/before-you-start/enforcement-notice');
+
+jest.mock('../../../../src/validators/before-you-start/enforcement-notice');
+
+describe('routes/before-you-start/enforcement-notice', () => {
+  beforeEach(() => {
+    // eslint-disable-next-line global-require
+    require('../../../../src/routes/before-you-start/enforcement-notice');
+  });
+
+  it('should define the expected routes', () => {
+    expect(get).toHaveBeenCalledWith(
```

Comment:
Isn't this functionality included in the use-a-different-notice ticket?

Reply:
Removed